src/main/java/org/jpeek/App.java

```diff
@@ -68,12 +68,12 @@
  * @checkstyle MethodLengthCheck (500 lines)
  * @checkstyle JavaNCSSCheck (500 lines)
  *
- * @todo #9:30min LCC metric has impediments (see puzzles in LCC.xml).
+ * @todo #9:30min LCC metric has impediments (see puzzles in LCC.xsl).
```

Comment:
I've seen this in 5 pull requests already. What's going on? Please, work in different branches. Here is the instruction: https://www.yegor256.com/2014/04/15/github-guidelines.html